nautobot/ipam/factory.py

```diff
@@ -248,7 +248,7 @@ class Params:
         has_role = NautobotBoolIterator()
         has_tenant = NautobotBoolIterator()
         has_vlan = NautobotBoolIterator()
-        has_vrf = NautobotBoolIterator()
+        # has_vrf = NautobotBoolIterator()
```

Comment:
Are we leaving this line here intentionally?

Edited: I see we are disabling `vrf` temporarily.

Reply:
Right it's going to be fixed in the namespace work since VRFs are unique to namespaces now.